config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -128,7 +128,8 @@ presubmits:
         - --provider=gce
         # Skipping snat tests probably GCE related? https://github.com/kubernetes/test-infra/issues/20321
         # Skipping Cloud Provider specific tests: LoadBalancer, ESIPP (Source IP Preservation)
-        - --test_args=--ginkgo.focus=\[sig-network\]|\[Conformance\]|\[Feature:NetworkPolicy\]|\[Feature:NetworkPolicyEndPort\] --ginkgo.skip=\[Feature:(Networking-IPv6|Example|Federation|PerformanceDNS|KubeProxyDaemonSetMigration|ServiceCIDRs|SCTPConnectivity)\]|DualStack|GCE|Disruptive|Serial|SNAT|LoadBalancer|ESIPP
+        # Skipping conntrack state invalid dropped packets metric test as Calico installs the drop rule before iptables and metric will never be updated. https://github.com/kubernetes/kubernetes/pull/122812
+        - --test_args=--ginkgo.focus=\[sig-network\]|\[Conformance\]|\[Feature:NetworkPolicy\]|\[Feature:NetworkPolicyEndPort\] --ginkgo.skip=\[Feature:(Networking-IPv6|Example|Federation|PerformanceDNS|KubeProxyDaemonSetMigration|ServiceCIDRs|SCTPConnectivity)\]|DualStack|GCE|Disruptive|Serial|SNAT|LoadBalancer|ESIPP|tcp_be_liberal
```

Comment:
I'm thinking maybe we should do this the other way around; add some `[Feature` tag or something so that the test only gets run where specifically requested, and then add that tag to the `ginkgo.focus` for the jobs that _should_ run it.

Because it's a pretty specialized test, and the calico failure just shows that we can't really expect it to work in arbitrary environments.

Reply:
We might need to add a new annotation [here](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/common/framework.go) and use that to define the test along with the specialised feature label as all the network tests are marked with `[sig-network]` label and will run by default even if don't pass out specialized feature label.

Reply:
this is already in the kube-proxy folder that is prefixed as `KubeProxy` so it should be used to select only kube-proxy specific tests https://github.com/kubernetes/kubernetes/pull/122812/files

We are trying to avoid again the explosion of labels, so it must be generic, if the feature identifies only one test  people has to end always skipping some tests based on special cases  so there is no much benefit

Reply:
It feels "unfair" to have a test that most people will have to skip...

Actually, I have a better idea: we already have a unit test for this metric, and an e2e test of the drop rule in general. The reason we wanted this e2e test as well was mostly to make sure that the nfacct code works correctly in real usage. But the nftables TODO proposes a few other nfacct-based metrics that would be easier to test in arbitrary clusters, like the one for localhost nodeports. So how about we drop the e2e test for the conntrack-dropped-packets metric, and then once we add the localhost nodeport metric, we can add an e2e test for that instead?

Reply:
I have dropped the e2e commit in the k/k PR for now.

Reply:
👍